### PR TITLE
Make plugin work for Ruby > 2.7

### PR DESCRIPTION
**Pull request by nadjaheitmann**

### CODE REVIEW

**Comment by nadjaheitmann** [comment]
@dosas What shall we do about failing Ruby 3.2 tests? Limit the version to Ruby 3.1? Any suggestions?

**Comment by dosas** [comment]
> @dosas What shall we do about failing Ruby 3.2 tests? Limit the version to Ruby 3.1? Any suggestions?

There are a lot of dependencies in the gemspec file that are pinned to quite some old versions, which should be adapted.

Updating rake to 12.0 fixes the problem for me. But I am not sure what happens to older versions then.

**Comment by nadjaheitmann** [comment]
I think it is a good idea to get the gemspec to use up-to-date versions

File: hammer_cli_foreman_scc_manager.gemspec [hunk 1]
```diff
@@ -17,12 +17,12 @@ Gem::Specification.new do |spec|
...
+  spec.add_dependency 'hammer_cli_foreman', '~> 3.10.0'
+  spec.required_ruby_version = '>= 2.7', '< 4'
 
   spec.add_development_dependency 'bundler', '< 2.5'
   spec.add_development_dependency 'minitest', '~> 5.0'
-  spec.add_development_dependency 'rake', '~> 10.0'
+  spec.add_development_dependency 'rake', '~> 12.0'
 
   spec.add_development_dependency 'rdoc', '~> 4'
   spec.add_development_dependency 'rubocop', '~> 1.6.1'
```
Comment (dosas) [inline hammer_cli_foreman_scc_manager.gemspec:28]: Out of scope for this PR but should be done in a follow up:

We should use the foreman rubocop and inherit its configuration.

**Review by dosas — APPROVED** [review]

**Pull request merged**